tools.rst

```diff
@@ -11,21 +14,48 @@ IATI is an open data standard and all internal IATI software tools are open sour
 - `IATI-Codelists-NonCore <https://github.com/IATI/IATI-Codelists-NonEmbedded>`__: codelists used by the IATI Standard, but not subjected to the same change-control process.
 - `IATI-Guidance <https://github.com/IATI/IATI-Guidance>`__: guidance and documentation that is not focused on IATI XML.
 - `IATI-Upgrades <https://github.com/IATI/IATI-Upgrades>`__: record of the process for and changes made to the IATI Standard over time.
-- `IATI-Standard-Website <https://github.com/IATI/IATI-Standard-Website>`__: Django and Wagtail website for IATI.
+
+
+Application layer Unified Platform repositories:
+------------------------------------------------
+
+- `IATI-refresher <https://github.com/IATI/refresher>`__: Django and Wagtail website for IATI.
+
+The IATI-refresher contains the applications that control the flow of the system; identifying, downloading and refreshing docs from the Registry, and then making sure they are validated and, if appropriate, subsequently indexed into the Datastore, using services provided by the repositories below.
+
+**Validation specific repositories:**
+
+- `IATI-Validator_API <https://github.com/IATI/js-validator-api>`__: Pure JavaScript IATI validator implementation.
+- `IATI-Validator_Services <https://github.com/IATI/validator-services>`__: Backend microservice provider for the IATI Validator on the Unified Platform.
+- `IATI-Validator_Web <https://github.com/IATI/IATI-Validator-Web>`__: IATI Validator Angular Front-end application.
+
+**Datstore specific repositories:**
```

Comment:
"Datastore specific repositories" - an **a** is missing

Reply:
Nice catch @andreaszenasidi , ran the whole thing through my spellchecker and this is the one obvious mistake it found.